### ISSUE TITLE
Raidboss - O12S triggers partly not working (though maybe act plugin issue?)

### ISSUE THREAD

**Issue by Ratsugo** [quisquous/cactbot#310]
A few weeks ago, after I reinstalled windows, act started to behave a bit wonky.
My issue is mostly noticable in o12s, while on m/f everything's still working fine, final omega has a few issues. 
Triggers for Diffuse Wavecannon and the Trigger for Archive Peripheral didn't seem to work anymore. 

The log get's filled with this error message on multiple occasions with the names of different players in the group:

`05.01.2019 04:03:01: Info: Cactbot_RaidBoss: BrowserConsole: Uncaught TypeError: Cannot set property 'Ratsugo Braddock' of undefined (Source: file:///C:/Program%20Files%20(x86)/Advanced%20Combat%20Tracker/OverlayPlugin/cactbot/ui/raidboss/raidboss.html, Line: 389)`

Tried quite a lot of things but still not sure what's causing it. I'm also having the issue that ACT doesn't show me the names of most of the attacks, instead it shows a number. As can be seen in this log:

https://www.fflogs.com/reports/dGwzpFbkDP6XJ327#fight=last&type=casts&hostility=1&source=2

The same happens with m/f as well, I just can't figure out what's causing this issue.

**Comment by quisquous** [comment]
No no, this is definitely a cactbot error and not an ffxiv_plugin error.

A few things to check.  On all your cactbot overlays, can you set the "dps update rate" to 0.  Also, make sure the high performance network filter is on for the ffxiv plugin.  Those are usually the causes of dropped logs more than anything.

Raidboss tries to track archive markers and hello world debuffs in o12s.  However, it resets them on the Ion Efflux cast (https://github.com/quisquous/cactbot/blob/master/ui/raidboss/data/triggers/o12s.js#L11).  If somehow none of those ever get hit, then you'll get errors elsewhere.  Can you look and see what it looks like when Omega casts Ion Efflux in your log?

**Comment by quisquous** [comment]
That is extremely weird to me.  The 14:3357 message should 100% say either `Unknown_3357` or `Ion Efflux` and not just `3357` on its own.  Is your FFXIV plugin up to date?

**Comment by Ratsugo** [comment]
Yeah, that's why I was assuming that it's caused by the FFXIV plugin. I'll probably see today with my group (haven't gotten to raid last week cause people were on vacation and I was being lazy) if it fixed with the new plugin and the settings I changed. 
It's weird that all the NPC skills in the instance are wrong like that in the ACT logs. Tried a lot of things, thought it might be caused by some other plugin but hadn't figured anything out. Also weird that it's not in all instances.

When zoning into 12s I get an error message about the Duty Recorder, so I even thought that might be the issue but I got that even before and the logs were still working at that time.

**Comment by quisquous** [comment]
The duty recorder is an in game thing, as is a red herring here.

Did things work in o12s this time around?

**Comment by Ratsugo** [comment]
Yeah everything working again now, I'm fairly certain it might've really just been the chat log settings. 
I saved them via the backup/restore tool, next time I'll just copy&paste them as I usually do lol

Issue can be closed then. Thank you very much for taking helping!
